master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -42,8 +42,9 @@ def __init__(self, url, port, user, password, db, captures,
         self.name = name
 
         self.captures = captures
-        self.client = InfluxDBClient(self.url, self.port, self.user,
-                                     self.password, self.db)
+        if InfluxDBClient is not None:
```

Comment:
@tardyp it's already done.
```
class InfluxStorageService(StatsStorageBase):
    """
    Delegates data to InfluxDB
    """

    def __init__(self, url, port, user, password, db, captures,
                 name="InfluxStorageService"):
        if not InfluxDBClient:
            config.error("Python client for InfluxDB not installed.")
        self.url = url
        self.port = port
        self.user = user
        self.password = password
        self.db = db
        self.name = name

        self.captures = captures
        self.client = InfluxDBClient(self.url, self.port, self.user,
                                     self.password, self.db)
        self._inited = True
```

my fix is about to not fail when we do: self.client = ...